src/ansys/grantami/recordlists/_connection.py

```diff
@@ -170,8 +171,63 @@ def get_list_items(self, record_list: RecordList) -> List[RecordListItem]:
             List of items included in the record list.
         """
         logger.info(f"Getting items in list {record_list} with connection {self}")
-        items = self.list_item_api.get_list_items(list_identifier=record_list.identifier)
-        return [RecordListItem._from_model(item) for item in items.items]
+        items_response = self.list_item_api.get_list_items(list_identifier=record_list.identifier)
+        return [RecordListItem._from_model(item) for item in items_response.items]
+
+    def get_resolvable_list_items(
+        self, record_list: RecordList, read_mode: bool = False
+    ) -> List[RecordListItem]:
+        """
+        Get all resolvable items included in a record list.
+
+        If an item cannot be resolved, it will not be returned. Performs multiple HTTP requests
+        against the Granta MI Server API.
+
+        Parameters
+        ----------
+        record_list : RecordList
+            Record list for which items will be fetched.
+        read_mode : bool
+            Whether to enable read-mode for users who ordinarily have write permissions. Has no
+            effect for read-only users.
+
+        Returns
+        -------
+        list of :class:`.RecordListItem`
+            List of items included in the record list.
+
+        Notes
+        -----
+        Whether an item can be resolved depends on the role the user has on the Granta MI server. As
+        a brief summary:
+
+        * If the item doesn't specify a version, this method tests if the user can access either the
+          record or, if in a version-controlled table, a version of the record in any state. The
+          resolution will not succeed if:
```

Comment:
`the resolution will not succeed`: just checking if I'm reading this right. In this context this means that the record is considered not resolvable, not that the method call will raise an error is that correct?

Reply:
That's correct, but if you're unsure that means users will also be unsure. I'll make it more explicit